src/main/java/com/google/sps/servlets/GetUsersServlet.java

```diff
@@ -91,7 +91,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 	private HashSet<String> queryUsersFromProjects(String username, String field, 
 			Datastore datastore) throws DatastoreException {
 		 // Query for projects where username == field
-		 Set<String> collaborators = new HashSet<>();
+		 HashSet<String> collaborators = new HashSet<>();
```

Comment:
Original was better practice, you want to use the interface. If in the future you would change to a different type of set, then everything would still work since they're all Sets.